common/web/types/src/util/util.ts

```diff
@@ -305,6 +305,15 @@ export function isPUA(ch: number) {
     (ch >= Uni_PUA_16_START && ch <= Uni_PUA_16_END));
 }
 
+/** @returns true if s is NEITHER NFC nor NFD */
+export function isDenormalized(s: string) : boolean {
```

Comment:
I'd prefer `isNormalized` because that way we avoid double-negatives :grin:

However, I see that will be tricky for `null` or `undefined`. There's a current assumption that passing `null` or `undefined` will return `false`, not denormalized. It might be better for those cases to throw, or else return the input?

Reply:
no, i think it would be better to return false (or true) here.  null, undefined, etc. aren't the exception case we are trying to find here. 

I will flip the sense of it. But basically, a falsy string is NOT the exception we're trying to catch.